Templates/Databases/MicrosoftSQL/Salesforce account synchronization to Microsoft SQL database/long-description.md

```diff
@@ -0,0 +1,21 @@
+This template synchronizes Salesforce account information with Microsoft SQL database table.
+Using this template, you can insert and update Salesforce account information to Microsoft SQL database on a schedule.
+
+![Template](assets/Salesforce account synchronization to Microsoft SQL database.svg)
```

Comment:
The screenshot of the table should not contain whitespaces, so maybe just replace spaces with underscores